src/client/java/teammates/client/scripts/DataMigrationForInstructorFeedbackResponseComments.java

```diff
@@ -0,0 +1,72 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.List;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+
+/**
+ * Script to set commentGiverType as INSTRUCTOR in all comments by instructor.
+ *
+ * <p>See issue #8830</p>
+ */
+
+public class DataMigrationForInstructorFeedbackResponseComments extends
```

Comment:
Please use the new data migration framework :) 

DataMigrationWithCheckpointForEntities

The old one cannot support large amount of entities.

Reply:
ohh Alright.